versions/v6/static/scss/components/_blockquotes.scss

```diff
@@ -0,0 +1,21 @@
+.blockquote {
```

Comment:
These styles are going to wind up conflicting with `.blockquote-quotation`/`.blockquote-quotation-reverse` if they're also added to the blockquote.

We work around this issue in Today by disabling the "Enable automatic insertion of opt-in content classes" setting in the Athena Shortcodes plugin (requires updating this line: https://github.com/UCF/Pegasus-Theme/blob/rc-v6.0.0/versions/v6/functions/config.php#L130), and by applying the default blockquote styles using the `blockquote:not(.blockquote)` selector (see https://github.com/UCF/Today-Child-Theme/blob/master/src/scss/mixins/_single-post-content.scss).

Reply:
Thanks! I've adjusted it here:
https://github.com/UCF/Pegasus-Theme/pull/100/commits/8c7c5eab7a86cebc22a7a117af378aeaec85c16c